src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Checkbox/Checkbox.spec.js

```diff
@@ -46,6 +47,7 @@ describe('Checkbox component', () => {
         innerCmp.simulate('change', null, true);
         expect(handleChange).toHaveBeenCalledWith('checkbox1', true);
         expect(fieldTouched).toHaveBeenCalledWith('checkbox1', true);
+        expect(props.onChange).toHaveBeenCalledWith(undefined, true);
```

Comment:
you can put initial data (value) instead of undefined

Reply:
Yes it's possible but we can have undefined as initial value, I added a test

Reply:
Great, thank you :D